src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -187,6 +202,11 @@ function BaseVideoPlayer({
             return;
         }
         shareVideoPlayerElements(videoPlayerRef.current, videoPlayerElementParentRef.current, videoPlayerElementRef.current, isUploading);
+        videoPlayerRef.current?.getStatusAsync().then((status) => {
+            if('rate' in status && status.rate) {
+                updatePlaybackSpeed(status.rate as PlaybackSpeed);
+            }
+        })
```

Comment:
Same, what case is this fixing?

Reply:
Assuming this test case.
1. Play `Video1`.
2. Then change playback speed to 0.25
3. Play `Video2`.
4. Then play `Video1` again.
5. Check playback speed (three dot menu -> playback speed).

Before fix, `Video1` is played in speed 0.25 but in menu it is set as 1(default value).